tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -442,10 +442,31 @@ func NewConfigSchedulerCommand() *cobra.Command {
 		newConfigHotRegionCommand(),
 		newConfigShuffleRegionCommand(),
 		newConfigGrantHotRegionCommand(),
+		newConfigBalanceLeaderCommand(),
 	)
 	return c
 }
 
+func newConfigBalanceLeaderCommand() *cobra.Command {
```

Comment:
The function is similar with `newConfigHotRegionCommand`,  maybe exract same code.

Reply:
Currently,  each scheduler has a different config API.  It is possible to refactor it in the future, so I think just implement it simply now.